src/sentry/conf/server.py

```diff
@@ -2746,5 +2746,7 @@ def build_cdc_postgres_init_db_volume(settings):
     "cache_name": "default",
 }
 
+SENTRY_FUNCTIONS_PROJECT_NAME = "hackweek-sentry-functions"
```

Comment:
@vpandiarajan20 I would leave it as `None` by default and set it in your own local config